hack/install-tools.sh

```diff
@@ -40,6 +41,13 @@ _install_gosec() {
 	_install_tool github.com/securego/gosec/v2/cmd/gosec@v2.13.1
 }
 
+_install_gitlint() {
+	_require_gobin
```

Comment:
nit: __require_gobin_ was called outside __install_ functions in all other places.